src/wlserver.cpp

```diff
@@ -1060,3 +1065,24 @@ void wlserver_surface_finish( struct wlserver_surface *surf )
 	wl_list_remove( &surf->pending_link );
 	wl_list_remove( &surf->destroy.link );
 }
+
+void wlserver_set_xwayland_server_mode( size_t idx, int w, int h, int refresh )
+{
+	gamescope_xwayland_server_t *server = wlserver_get_xwayland_server( idx );
+	if ( !server )
+		return;
+	wl_client *client = server->get_client();
+
+	struct wl_resource *resource;
+	wl_resource_for_each( resource, &wlserver.wlr.output->resources )
+	{
+		struct wlr_output *output = wlr_output_from_resource( resource );
```

Comment:
This is the same as `wlserver.wlr.output`

Reply:
Fixed.